backbone.blazer.js

```diff
@@ -58,6 +58,7 @@ Backbone.Blazer.Router = Backbone.Router.extend({
         var router = this;
 
         route.trigger('before:execute', routeData);
+        router.trigger('before:execute', routeData);
```

Comment:
cool. since this is being fired on the router now, should routeData include the route name/pattern?